public/src/components/channelManagement/campaigns/TestCard.tsx

```diff
@@ -189,7 +189,7 @@ function TestCard({ test, keyId, linkPath }: TestCardProps): React.ReactElement
   };
 
   const getLocations = (test: Test) => {
-    const locations: string[] = test.locations || [];
+    const locations: (string | number)[] = test.locations || [];
```

Comment:
Oh! When might this be a `number`?

Reply:
The change fixes a typescript error. Given that the campaigns page functionality is currently broken due to us having to separate out archived tests into their own DynamoDB table, and I don't think Marketing are using the Campaigns functionality as we expected them to, thus needs review, I think we can live with this in the short-term?

Reply:
Sure thing, if you're happy that's all good! (I'm still not sure when `test.locations` will include a `number`, but if it's a short term thing I'm sure we can live with it).